libs/deps/commerce.js

```diff
@@ -1,6 +1,5 @@
-  //# sourceMappingURL=commerce.js.map
-  
\ No newline at end of file
+// branch: develop commit: b3f6608faa10db8d0187b310044d4690d063f1bf Thu, 06 Jun 2024 16:47:00 GMT
```

Comment:
@yesil - I see this file generated from different branches and I'm wondering if there's any risk of introducing unwanted changes. Should the file be built from an official branch like `main` after everything has been properly tested? cc @npeltier , @3ch023

Reply:
Narcis, I opened this revert PR via github, and it should normally revert to previous good version of commerce.js that was in prod.